docs/dump_javadump.md

```diff
@@ -27,8 +27,635 @@
+```
+
+## HOOKS
+
+This section shows internal VM event callbacks, which are used for diagnosing performance problems in the VM. Multiple hook interfaces are listed, which include their individual hook events. The following example shows data for the `J9VMHookInterface`, including the call site location (source file:line number), start time, and duration of the last callback and the longest callback.
+
+```
+NULL           ------------------------------------------------------------------------
+0SECTION       HOOK subcomponent dump routine
+NULL           ==============================
+1HKINTERFACE   MM_OMRHookInterface
+NULL           ------------------------------------------------------------------------
+1HKINTERFACE   MM_PrivateHookInterface
+NULL           ------------------------------------------------------------------------
+1HKINTERFACE   MM_HookInterface
+NULL           ------------------------------------------------------------------------
+1HKINTERFACE   J9VMHookInterface
+NULL           ------------------------------------------------------------------------
+2HKEVENTID     1
+3HKCALLCOUNT       18
+3HKLAST            Last Callback
+4HKCALLSITE           trcengine.c:392
+4HKSTARTTIME          Start Time: 2018-08-30T21:55:47.601
+4HKDURATION           DurationMs: 0
+3HKLONGST          Longest Callback
+4HKCALLSITE           trcengine.c:392
+4HKSTARTTIME          Start Time: 2018-08-30T21:55:47.460
+4HKDURATION           DurationMs: 1
+NULL
+...
+1HKINTERFACE   J9VMZipCachePoolHookInterface
+NULL           ------------------------------------------------------------------------
+1HKINTERFACE   J9JITHookInterface
+NULL           ------------------------------------------------------------------------
+2HKEVENTID     3
+3HKCALLCOUNT       65
+3HKLAST            Last Callback
+4HKCALLSITE           ../common/mgmtinit.c:191
+4HKSTARTTIME          Start Time: 2018-08-30T21:55:47.601
+4HKDURATION           DurationMs: 0
+3HKLONGST          Longest Callback
+4HKCALLSITE           ../common/mgmtinit.c:191
+4HKSTARTTIME          Start Time: 2018-08-30T21:55:47.486
+4HKDURATION           DurationMs: 0
+...
+NULL
+```
+
+## SHARED CLASSES
+
+If the shared classes cache is enabled at run time, the information provided in a Java dump file describes settings that were used when creating the cache, together with summary information about the size and content of the cache.
+
+In the following example, the shared classes cache was created with a Class Debug Area (`-Xnolinenumbers=false`). Byte code instrumentation (BCI) is enabled, which is the default, and VMs sharing the cache are allowed to store classpaths, which is also the default.
+
+The `Cache Summary` shows a cache size (`2SCLTEXTCSZ`) of 16776608 bytes, with a soft maximum size (`2SCLTEXTSMB`) also of 16776608 bytes, which leaves 12691668 bytes of free space (`2SCLTEXTFRB`). The size of the Class Debug Area (`2SCLTEXTDAS`) is 1331200 bytes, which uses only 11% of the available space.
```

Comment:
11% is the amount used inside of the class debug area, i.e. 156240 is used out of 1331200.

Reply:
Fixed, thanks.